core/org.wso2.carbon.bootstrap/src/test/java/org/wso2/carbon/bootstrap/CheckinClientBootstrapTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2017 WSO2, Inc. (http://wso2.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.bootstrap;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.nio.file.Paths;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.AssertJUnit.assertNotNull;
+import static org.wso2.carbon.bootstrap.Bootstrap.ROOT;
+
+/**
+ * Tests functionality of CheckinClientBootstrap class
+ */
+public class CheckinClientBootstrapTest {
+    private CheckinClientBootstrap checkinClientBootstrap;
+
+    /**
+     * Create an instance of CheckinClientBootstrap
+     */
+    @BeforeMethod
+    public void setUp() {
+        checkinClientBootstrap = new CheckinClientBootstrap();
+    }
+
+    /**
+     * Test if the file path assigning is successful
+     *
+     * @throws MalformedURLException
```

Comment:
Shall we add descriptions on the exception?

Reply:
Fixed in cc1487d027afcdd325678ef87fbc5f77607b7171